tests/acceptance/bootstrap/FilesVersionsContext.php

```diff
@@ -36,6 +37,27 @@
  */
 class FilesVersionsContext implements Context {
 	private FeatureContext $featureContext;
+	private SpacesContext $spacesContext;
+
+	/**
+	 * This will run before EVERY scenario.
+	 * It will set the properties for this object.
+	 *
+	 * @BeforeScenario
+	 *
+	 * @param BeforeScenarioScope $scope
+	 *
+	 * @return void
+	 */
+	public function before(BeforeScenarioScope $scope):void {
+		// Get the environment
+		$environment = $scope->getEnvironment();
+		// Get all the contexts you need in this context
+		$this->featureContext = BehatHelper::getContext($scope, $environment, 'FeatureContext');
+		if (!OcisHelper::isTestingOnReva()) {
+			$this->spacesContext = $environment->getContext('SpacesContext');
+		}
```

Comment:
```suggestion
		$this->spacesContext = BehatHelper::getContext($scope, $environment, 'SpacesContext');
```

Reply:
why are you so confused about this code? Please be sure what you want to review or please add proper reason.